PyTorch/SpeechRecognition/Jasper/Dockerfile

```diff
@@ -15,14 +15,33 @@
 ARG FROM_IMAGE_NAME=nvcr.io/nvidia/pytorch:19.10-py3  
 FROM ${FROM_IMAGE_NAME}
 
+WORKDIR /workspace/jasper
+# Install requirements (do this first for better caching)
+# Copy rest of files
+COPY requirements.txt .
+COPY trt/requirements.txt ./trt_requirements.txt
+RUN pip install --disable-pip-version-check -U -r requirements.txt -r trt_requirements.txt
+
+COPY ./trt/onnx-trt.patch .
```

Comment:
please do not change this Dockerfile. We have different Dockfiles for different purposes. This Dockerfile is for pure PyTorch training of Jasper
Everything TRT-relased is in the Dockerfile in  PyTorch/SpeechRecognition/Jasper/TRT/Dockerfile